src/components/Login/Login.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import axios from 'axios';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: '',
+    };
+    this.onChange = this.onChange.bind(this);
+    this.submitForm = this.submitForm.bind(this);
+  }
+
+  onChange(e) {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  }
+
+  submitForm(e) {
+    e.preventDefault();
+    const url = 'http://localhost:8000/users/';
```

Comment:
login should post to a login path, something like "/login". posting to "/users" should create a user, not login